src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js

```diff
@@ -176,6 +179,9 @@ export class CernerCallToAction extends Component {
               cernerFacility => cernerFacility?.facilityId === strippedID,
             );
 
+            // Derive the link text/label.
+            const linkText = isCerner ? 'Go to My VA Health' : myHealtheVetLink;
+
```

Comment:
i think we meant `myHealtheVetLinkText` here instead of `myHealtheVetLink`?

Reply:
Oh true!